src/Forms/HTMLEditor/TinyMCEConfig.php

```diff
@@ -676,4 +680,37 @@ public static function get_tinymce_lang()
         }
         return 'en';
     }
+
+    /**
+     * @return string|false
+     */
+    public function getAdminPath()
+    {
+        $module = ModuleLoader::instance()->getManifest()->getModule('silverstripe/admin');
+        if ($module) {
+            return $module->getRelativePath();
+        }
+        return false;
+    }
+
+    /**
+     * @return string|false
+     */
+    public function getTinyMCEPath()
+    {
+        $configDir = static::config()->get('base_dir');
+        if ($configDir) {
+            return $configDir;
+        }
+
+        if ($this->getAdminPath()) {
```

Comment:
getAdminPath() will be `""` (empty string) if the admin is the root module.

Instead of getting the admin path, get the admin module and use the resource API to get the nested path.

```
if ($admin = $this->getAdminModule()) {
	return $admin->getResourcePath('thirdparty/tinymce');
}
```